.github/workflows/testBuildHybrid.yml

```diff
@@ -5,52 +5,45 @@ on:
     inputs:
       PULL_REQUEST_NUMBER:
         description: Pull Request number for correct placement of apps
-        required: true
-  pull_request_target:
-    types: [opened, synchronize, labeled]
-    branches: ['*ci-test/**']
-
-env:
-  PULL_REQUEST_NUMBER: ${{ github.event.number || github.event.inputs.PULL_REQUEST_NUMBER }}
+        required: false
+        default: ''
+      OLD_DOT_PULL_REQUEST_NUMBER:
+        description: Old Dot Pull request number to build the app from. It will take precedence over MOBILE-EXPENSIFY from ND PR description if both specified. 
+        required: false
+        default: ''
 
 jobs:
   prep:
     runs-on: ubuntu-latest
-    outputs:
-      REF: ${{ github.event.pull_request.head.sha || steps.getHeadRef.outputs.REF }}
-      READY_TO_BUILD: ${{ steps.readyToBuild.outputs.READY_TO_BUILD }}
     steps:
-      - name: Checkout
-        if: ${{ github.event_name == 'workflow_dispatch' }}
-        uses: actions/checkout@v4
-
       - name: Validate that user is an Expensify employee
         uses: ./.github/actions/composite/validateActor
         with:
           REQUIRE_APP_DEPLOYER: false
           OS_BOTIFY_TOKEN: ${{ secrets.OS_BOTIFY_COMMIT_TOKEN }}
 
-      - name: Check if PR has Ready to Build label
-        id: readyToBuild
+      - name: Validate input
         run: |
-          LABELS=$(gh pr view "${{ env.PULL_REQUEST_NUMBER }}" --repo Expensify/App --json labels --jq '.labels[].name')
-          if echo "$LABELS" | grep -q 'Ready To Build'; then
-            echo "::notice::✅ PR ${{ env.PULL_REQUEST_NUMBER }} has 'Ready to Build' label"
-            echo "READY_TO_BUILD=true" >> "$GITHUB_OUTPUT"
-          else
-            echo "::error::❌ PR ${{ env.PULL_REQUEST_NUMBER }} does not have 'Ready to Build' label"
-            echo "READY_TO_BUILD=false" >> "$GITHUB_OUTPUT"
+          if [[ -z "${{ github.event.inputs.PULL_REQUEST_NUMBER }}" && -z "${{ github.event.inputs.OLD_DOT_PULL_REQUEST_NUMBER }}" ]]; then
+            echo "Invalid input. You have to pass at least one PR number"
```

Comment:
I guess this is the workaround because Github doesn't allow us to specify that one of the param fields must be passed.

Reply:
I think that is totally fine, just want to check

Reply:
Yeah, to it's to be sure at least one PR is added so we have place to put comments.

Reply:
> And we're still using PR number rather than URL for now?
Now two options are valid. You as internal can add OD PR number while dispatching workflow and we as externals can add link to ND PR description. The one passed during dispatch has higher priority. 
I think it's handy because OD PR can be used to trigger build when there are only changes in Mobile-Expensify. And if some developer works on changes that require both ND and OD then they can link PRs and internals don't have to remember to pass two PRs when dispatching workflow